### PR TITLE
Add Tales of the Abyss

### PR DESCRIPTION
**Pull request by jdenm8**
Adds Tales of the Abyss.

*Automatically generated by PR builder*

### CODE REVIEW

File: tales-of-the-abyss/game.dat [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+[[testcases]]
+title = "000400000005D700"
+compatibility = "3"
+date = "2017-07-12"
+version = "HEAD-c017065"
+cpu = "Intel i5 2500K"
+gpu = "AMD Radeon R9 380"
+os = "Windows 10 Build 14393"
+author = "jdenm8"
```
Comment (TheKoopaKingdom) [inline tales-of-the-abyss/game.dat]: Usually `author` goes directly under `version`.

**Comment by jdenm8** [comment]
Must be something wrong with the game submission form then, it's what created the folder. 

I'll figure out how to fix the folder on the binary files.

**Comment by jdenm8** [comment]
That should be everything fixed.

**Comment by TheKoopaKingdom** [comment]
LGTM.

**Pull request merged**